tests/test_writing_api.py

```diff
@@ -199,4 +200,29 @@ def test_update_judgment_fail(mocked_client):
         "is-a-judgment/uri", b"<judgment></judgment>", ""
     )
     assert response.status_code == 409
-    assert "request needed a checkout first" in response.text
+    assert (
+        response.text
+        == '{"detail":"The resource is not checked out by anyone, but that request needed a checkout first."}'
+    )
+
+
+@patch("openapi_server.apis.writing_api.client_for_basic_auth")
+def test_judgment_validation_fail(mocked_client):
+    error = MarklogicValidationFailedError()
+    error.response = Mock()
+    error.response.content = b'<error-response xmlns="http://marklogic.com/xdmp/error"><message>a message</message></error-response>'  # noqa:E501
+    # Checkout Judgment fails with an error or returns None
```

Comment:
We can remove this comment right, I think it is just a leftover from other tests?